simplivity/resources/policies.py

```diff
@@ -73,6 +73,23 @@ def get_by_data(self, data):
         """
         return Policy(self._connection, self._client, data)
 
+    def create(self, name, flags=None, timeout=-1):
+        """ create the new policy
+
+        Args:
+            flags: Dictionary of filters, example: {'cluster_group_id': 'cluster_group_id'}
+            name : name of policy
+            timeout : timeout for the operation to complete
+
+        Returns:
+            Returns Policies object, list of policy which will always have single
```

Comment:
Update the docstring since we will never return a list of policies

i.e.

        Returns:
            object: Policy object.

Reply:
Updated string as per suggestion